packages/seo-plugin/src/seo-interface/analysis/components/Analysis.vue

```diff
@@ -0,0 +1,414 @@
+		icon: 'error',
+		iconClass: 'error',
+		results: problemResults.value,
+	},
+	{
+		id: 'improvements' as SectionId,
+		title: 'Improvements',
+		icon: 'warning',
+		iconClass: 'warning',
+		results: improvementResults.value,
+	},
+	{
+		id: 'good' as SectionId,
+		title: 'Good results',
+		icon: 'check_circle',
+		iconClass: 'good',
+		results: goodResults.value,
+	},
+]);
+
+const availableSectionIds = computed(() => sections.value.filter((s) => s.results.length > 0).map((s) => s.id));
+
+function expandAllSections() {
+	openSectionIds.value = availableSectionIds.value;
+}
+
+function collapseAllSections() {
+	openSectionIds.value = [];
+}
+
+const hasOnlyNeutralResults = computed(() => {
+	return allAnalyses.value.length > 0 && availableSectionIds.value.length === 0;
+});
+
+const canCollapseAll = computed(() => openSectionIds.value.length > 0);
+const canExpandAll = computed(() => openSectionIds.value.length < availableSectionIds.value.length);
+</script>
+
+<template>
+	<div class="seo-analysis-container field">
+		<div class="header">
+			<label class="label field-label type-label">Analysis</label>
+			<div class="action-bar">
+				<v-button
+					v-tooltip="'Collapse all'"
+					:disabled="!canCollapseAll"
+					icon
+					small
+					secondary
+					title="Collapse all"
+					@click="collapseAllSections"
+				>
+					<v-icon name="unfold_less" />
+				</v-button>
+
+				<v-button
+					v-tooltip="'Expand all'"
+					:disabled="!canExpandAll"
+					icon
+					small
+					secondary
+					title="Expand all"
+					@click="expandAllSections"
+				>
+					<v-icon name="unfold_more" />
+				</v-button>
+			</div>
+		</div>
+
+		<div v-if="!focusKeyphrase" class="empty-state">
+			Enter a focus keyphrase to analyze your content.
+		</div>
+
+		<template v-else>
+			<AccordionRoot
+				v-model="openSectionIds"
+				type="multiple"
+				collapsible
+				class="analysis-accordion"
+				:unmount-on-hide="false"
+			>
+				<template v-for="section in sections" :key="section.id">
+					<AccordionItem
+						v-if="section.results.length > 0"
+						:value="section.id"
+						class="accordion-item"
+					>
+						<AccordionHeader class="accordion-header">
+							<AccordionTrigger class="accordion-trigger">
+								<div class="section-title" :class="section.id">
+									<v-icon :name="section.icon" :class="section.iconClass" />
+									{{ section.title }} ({{ section.results.length }})
+								</div>
+								<v-icon name="chevron_right" class="section-icon" />
+							</AccordionTrigger>
+						</AccordionHeader>
+						<AccordionContent class="accordion-content">
+							<AnalysisResultComponent
+								v-for="result in section.results"
+								:key="result.id"
+								:result="result"
+							/>
+						</AccordionContent>
+					</AccordionItem>
+				</template>
+			</AccordionRoot>
+
+			<div v-if="hasOnlyNeutralResults" class="all-neutral-state">
+				All analysis checks are neutral. This might happen if content is missing or too short.
```

Comment:
Instead of `All analysis checks are neutral. This might happen if content is missing or too short.`, I would say something like `We found nothing to report! This might happen if content is missing or too short.`. Idk what `neutral` means in this context.